### ISSUE TITLE
default docker image sklearn version

### ISSUE THREAD

**Issue by ihsaan-ullah** [codalab/codabench#801]
In the fefault docker image `codalab/codalab-legacy:py3`, balanced accuracy score import fails

```
from sklearn.metrics import balanced_accuracy_score
```

<img width="795" alt="Screenshot 2023-04-07 at 5 53 58 PM" src="https://user-images.githubusercontent.com/13259262/230612180-81d8da37-3963-4206-b57f-ffa52dfad27a.png">

**Comment by ihsaan-ullah** [comment]
Yes I am using this in Fair-Universe competition. I think this should be there in the basic docker image because this is one of the basic metrics in sklearn. What do you think

**Comment by Didayolo** [comment]
Solved by:

- #807
